content/en/blog/_posts/2019-05-22-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

```diff
@@ -0,0 +1,178 @@
+
+---
+layout: blog
+title: "OPA Gatekeeper: Policy and Governance for Kubernetes"
+date: 2019-05-22
+slug: OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes 
+---
+
+**Authors:** Rita Zhang (Microsoft)
+
+In this post, we will walkthrough the goals, history, and current state of the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) project, and how you can leverage it to enforce policies and strengthen governance in your Kubernetes environment. 
+
+You can also check out recordings from the following Kubecon EU 2019 sessions:
+* [Intro: Open Policy Agent Gatekeeper](https://sched.co/MPiM)
+* [Deep Dive: Open Policy Agent](https://sched.co/MPk0)
+
+## Motivations 
+
+If your organization has been operating Kubernetes, you probably have been looking for ways to control what end-users can do on the cluster and ways to ensure the clusters are always in compliance with company policies. These policies may be there to meet governance and legal requirements or to enforce best practices and organizational conventions. With Kubernetes, how do we ensure compliance without sacrificing development agility and operation independence? 
```

Comment:
"the clusters are always in compliance" -> "that clusters are in compliance" ( change "the" to "that", drop "always")